### ISSUE TITLE
Webhook help not working

### ISSUE THREAD

**Issue by AdminRAT** [discordconsole-team/DiscordConsole#43]
Whenever I try to access the help, it outputs an error
```> help
DiscordConsole has crashed.
Please tell the DiscordConsole Dev team what you did to cause this.
https://mnpn.me/contact, https://discord.gg/xvQV8bT
Error Details: runtime error: invalid memory address or nil pointer dereference```

**Comment by Mnpn03** [comment]
Very odd.

**Comment by Mnpn03** [comment]
Resolved in 5e37f6f [2.4.2]. Thanks!
